src/lib/InMemoryDataSource.js

```diff
@@ -0,0 +1,34 @@
+const Datasource = require('../Datasource');
```

Comment:
This file should be under `./datasource` folder

Reply:
done